src/app/lib/utilities/srcSet/index.js

```diff
@@ -30,8 +30,12 @@ export const createSrcset = (
 
   return urls.join(', ');
 };
-export const getPlaceholderSrcSet = ({ originCode, locator }) =>
-  DEFAULT_RESOLUTIONS.map(
+export const getPlaceholderSrcSet = ({ originCode, locator }) => {
+  if (!originCode || !locator) {
```

Comment:
Could this be `if(!originCode || locator))`

Reply:
Nope, the logic is different.

If only one of mine has a value, it will hit line 35

If only one of yours has a value, it will not hit line 35, meaning it will try to run the following code with undefined values.

Reply:
Aye these two expressions are different - updated it to make the intent clearer